UI/data/themes/Rachni.qss

```diff
@@ -0,0 +1,1032 @@
+/***************************************************************************/
+/*   Copyright (C) 2017-2018 by Joel Bethke <joel.bethke@gmail.com>        */
+/*                                                                         */
+/*                                                                         */
+/*   This program is free software: you can redistribute it and/or modify  */
+/*   it under the terms of the GNU General Public License as published by  */
+/*   the Free Software Foundation, either version 2 of the License, or     */
+/*   (at your option) any later version.                                   */
```

Comment:
That's correct. If this is GPLv2, then it's compatible with OBS's GPLv2 license. This is fine, as far as I can tell.